tutorials/monitoring-demo/walkthrough-setup.md

```diff
@@ -23,16 +23,16 @@ Once you have downloaded the zip package make sure to **unblock** it in the file
 
 After unblocking the zip file, extract its contents into a folder. For the rest of this tutorial, we will refer to this folder as `MonitoringDemo`.
 
-Open the `MonitoringDemo` folder and double-click on `run.bat`. This script will:
+Open the `MonitoringDemo` folder and double-click on `MonitoringDemo.exe`. This executable will:
 
-1. You will be prompted to run the remaining steps with elevated privileges. This is needed so that processes can bind to their network ports.
```

Comment:
This part was a leftover that we apparently did not fix when we worked on the platform package. Now making sure it is up to date